src/igvfd/tests/data/inserts/user.json

```diff
@@ -308,13 +308,1417 @@
+        "last_name": "Roth",
+        "lab": "/labs/frederick-roth/",
+        "viewing_groups": [
+            "community"
+        ],
+        "job_title": "PI",
+        "groups": [
+            "verified"
+        ],
+        "submits_for": [
+            "/labs/frederick-roth/"
+        ],
+        "status": "current",
+        "uuid": "00b5591d-1e3d-41bf-be82-13a218283ada"
+    },
+    {
+        "email": "mikko@email_domain.com",
+        "first_name": "Mikko",
+        "last_name": "Taipale",
+        "lab": "/labs/mikko-taipale/",
+        "viewing_groups": [
+            "community"
+        ],
+        "job_title": "PI",
+        "groups": [
+            "verified"
+        ],
+        "submits_for": [
+            "/labs/mikko-taipale/"
+        ],
+        "status": "current",
+        "uuid": "569575e3-67d6-4573-9301-c2243354bda2"
+    },
+    {
+        "email": "carpenter@email_domain.com",
+        "first_name": "Anne",
+        "last_name": "Carpenter",
+        "lab": "/labs/anne-carpenter/",
+        "viewing_groups": [
+            "community"
+        ],
+        "job_title": "PI",
+        "groups": [
+            "verified"
+        ],
+        "submits_for": [
+            "/labs/anne-carpenter/"
+        ],
+        "status": "current",
+        "uuid": "ad7e8437-3dff-46af-ac35-d7ccb1f58aee"
+    },
+    {
+        "email": "hyejung@email_domain.com",
+        "first_name": "Hyejung",
+        "last_name": "Won",
+        "lab": "/labs/hyejung-won/",
+        "viewing_groups": [
+            "community"
+        ],
+        "job_title": "PI",
+        "groups": [
+            "verified"
+        ],
+        "submits_for": [
+            "/labs/hyejung-won/"
+        ],
+        "status": "current",
+        "uuid": "7e51864b-2e2b-40cf-9abc-5cc2dc98f35d"
+    },
+    {
+        "email": "karen@email_domain.com",
+        "first_name": "Karen",
+        "last_name": "Mohlke",
+        "lab": "/labs/karen-mohlke/",
+        "viewing_groups": [
+            "community"
+        ],
+        "job_title": "PI",
+        "groups": [
+            "verified"
+        ],
+        "submits_for": [
+            "/labs/karen-mohlke/"
+        ],
+        "status": "current",
+        "uuid": "3e3409dd-e4a8-4b7c-bf31-22d2b089ae1f"
+    },
+    {
+        "email": "michael@email_domain.com",
+        "first_name": "Michael",
+        "last_name": "Love",
+        "lab": "/labs/michael-love/",
+        "viewing_groups": [
+            "community"
+        ],
+        "job_title": "PI",
+        "groups": [
+            "verified"
+        ],
+        "submits_for": [
+            "/labs/michael-love/"
+        ],
+        "status": "current",
+        "uuid": "3a0ff5da-c893-4e36-9d0e-d97940a7e13d"
+    },
+    {
+        "email": "buenrostro@email_domain.com",
+        "first_name": "Jason",
+        "last_name": "Buenrostro",
+        "lab": "/labs/jason-buenrostro/",
+        "viewing_groups": [
+            "community"
```

Comment:
Shouldn't the default viewing group for the majority of these people be IGVF not community?